src/toil_scripts/exome_variant_pipeline/exome_variant_pipeline.py

```diff
@@ -20,7 +20,7 @@
 from toil_scripts.tools.mutation_callers import run_muse
 from toil_scripts.tools.mutation_callers import run_mutect
 from toil_scripts.tools.mutation_callers import run_pindel
-from toil_scripts.tools.preprocessing import run_gatk_preprocessing
+from toil_scripts.tools.preprocessing import run_preprocessing
```

Comment:
The exome pipeline was using a version of preprocessing that did not include MarkDuplicates, but the version germline uses does. I was calling the germline preprocessing function run_gatk_preprocessing.